CMakeLists.txt

```diff
@@ -276,7 +269,7 @@ elseif(DS2_ARCHITECTURE STREQUAL RISCV)
 endif()
 
 # OS Sources
-if(CMAKE_SYSTEM_NAME MATCHES "Windows")
+if(WIN32)
```

Comment:
Does WIN32 cover the OS_NAME=WindowsStore case?

Reply:
No, that only covers Windows; this is a good point - this should be `WIN32 OR WINDOWS_STORE`.